service/src/main/resources/api/openapi.yml

```diff
@@ -5,35 +5,86 @@ info:
     Terra Axon Server
 
   version: 0.0.1
+
+security:
+- authorization: [ openiAd, email, profile ]
+- bearerAuth: [ ]
+
 paths:
   # Unauthenticated Paths
   /status:
     get:
-      security: [ ]
+      security: []
       summary: Returns the operational status of the service
       operationId: serviceStatus
-      tags: [ Public ]
+      tags: [Public]
       responses:
-        '200':
+        "200":
```

Comment:
Hmm... why use double quotes? I don't think it matters. Just curious.

Reply:
IDE yaml formatter, I left it as is but I could try to configure it to use single quotes

Reply:
No need. I haven't used the IDE formatter. Sounds nice